README.md

```diff
@@ -36,6 +36,7 @@ In the following, a list of all supported languages with their supported languag
 | [EMF Metamodel](https://www.eclipse.org/modeling/emf/) |  2.25.0 | emf               |                                beta                                 |    EMF    |
 | [EMF Model](https://www.eclipse.org/modeling/emf/)     |  2.25.0 | emf-model         |                                alpha                                |    EMF    |
 | [LLVM IR](https://llvm.org)                            |      15 | llvmir            |                                beta                                 |  ANTLR 4  |
+| [TypeScript](https://www.typescriptlang.org/)          |      ~5 | typescript        |                                beta                                 |  ANTLR 4  |
```

Comment:
Add a hint for javascript support and maybe add a link at the ~5 to the ANTLR readme that discusses what is supported?